libsolidity/analysis/ViewPureChecker.cpp

```diff
@@ -368,7 +368,6 @@ void ViewPureChecker::endVisit(MemberAccess const& _memberAccess)
 			{MagicType::Kind::ABI, "encodePacked"},
 			{MagicType::Kind::ABI, "encodeWithSelector"},
 			{MagicType::Kind::ABI, "encodeWithSignature"},
-			{MagicType::Kind::Block, "blockhash"},
```

Comment:
Should be mentioned in the changelog.

Reply:
This is dead code. Can remove from this PR and deal with this sperately.

Reply:
Ah sorry!